packages/input-container/theme/lumo/vaadin-input-container-styles.js

```diff
@@ -8,14 +8,18 @@ registerStyles(
   'vaadin-input-container',
   css`
     :host {
-      border-radius: var(--lumo-border-radius-m);
+      border-start-start-radius: var(--vaadin-input-container-top-start-radius, var(--_input-container-radius));
+      border-start-end-radius: var(--vaadin-input-container-top-end-radius, var(--_input-container-radius));
+      border-end-start-radius: var(--vaadin-input-container-bottom-start-radius, var(--_input-container-radius));
+      border-end-end-radius: var(--vaadin-input-container-bottom-end-radius, var(--_input-container-radius));
       background-color: var(--lumo-contrast-10pct);
-      padding: 0 calc(0.375em + var(--lumo-border-radius-m) / 4 - 1px);
+      padding: 0 calc(0.375em + var(--_input-container-radius) / 4 - 1px);
       font-weight: 500;
       line-height: 1;
       position: relative;
       cursor: text;
       box-sizing: border-box;
+      --_input-container-radius: var(--lumo-border-radius-m);
```

Comment:
Should this have `--vaadin-input-container-border-radius` as the primary value?

Reply:
@jouni WDYT, should we add a common custom property in addition to separate properties?

Reply:
Yes, that’s probably the one developers would like to use 99% of the time (have a unified radius for all corners).